Source: Debian
Upstream: No, upstream is archived

From: Andreas Henriksson <andreas@fatal.se>
Date: Mon, 15 Aug 2016 18:12:06 +0200
Subject: [PATCH] daemon: fix conflicting return types

... in functions seahorse_dbus_server_{init,cleanup}

Reported-by: Michael Tautschnig <mt@debian.org>
Bug-Debian: https://bugs.debian.org/749001

https://bugzilla.gnome.org/show_bug.cgi?id=769943
(cherry picked from commit a576e407d913b35804f838d615da52899681d61d)
---
 daemon/seahorse-daemon.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/daemon/seahorse-daemon.h b/daemon/seahorse-daemon.h
index 93c0c45..09716bb 100644
--- a/daemon/seahorse-daemon.h
+++ b/daemon/seahorse-daemon.h
@@ -24,9 +24,9 @@
 /* seahorse-dbus-server.c --------------------------------------------------- */
 
 
-gboolean            seahorse_dbus_server_init ();
+void                seahorse_dbus_server_init ();
 
-gboolean            seahorse_dbus_server_cleanup ();
+void                seahorse_dbus_server_cleanup ();
 
 DBusGConnection*    seahorse_dbus_server_get_connection ();
 
